internal/ptr/ptr.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
We tend to avoid [top level util packages](https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common) and sign up to every package having at least some scoped purpose.